unocss/package.json

```
@@ -0,0 +1,35 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "run-s \"build:*\"",
+    "build:remix": "remix build",
+    "build:css": "unocss './app/**/*.{js,ts,jsx,tsx}' -o ./app/styles/uno.css",
+    "dev": "run-p \"dev:*\"",
+    "dev:remix": "remix dev",
+    "dev:css": "unocss './app/**/*.{js,ts,jsx,tsx}' -o ./app/styles/uno.css --watch",
+    "start": "remix-serve build"
+  },
```

What about having a `generate:css` script as well, like we have in the `tailwindcss` example?

```suggestion
  "scripts": {
    "build": "run-s \"build:*\"", 
    "build:css": "npm run generate:css", 
    "build:remix": "remix build", 
    "dev": "run-p \"dev:*\"", 
    "dev:css": "npm run generate:css -- --watch", 
    "dev:remix": "remix dev", 
    "generate:css": "unocss './app/**/*.{js,ts,jsx,tsx}' -o ./app/styles/uno.css", 
    "start": "remix-serve build"
  },
```

---

It's just adding even more complexity in my opinion. But sure I'll add it.

---

Done